libs/blocks/global-navigation/global-navigation.js

```diff
@@ -608,9 +608,6 @@ class Gnav {
       locale,
       imsClientId: window.adobeid?.client_id,
       theme: isDarkMode() ? 'dark' : 'light',
-      onReady: () => {
-        this.decorateAppPrompt({ getAnchorState: () => window.UniversalNav.getComponent?.('app-switcher') });
-      },
```

Comment:
Is `onReady` callback removed in new version?

Reply:
As mentioned in PR description - Unav suggested to not rely on onReady

Reply:
and this is true for 1.2 as well?

Reply:
i don't think it affects 1.2 as the window.UNav() promise came up with 1.3